rest/rest-server/src/main/java/org/apache/brooklyn/rest/resources/SensorResource.java

```diff
@@ -140,6 +167,11 @@ public void set(String application, String entityToken, String sensorName, Objec
     @Override
     public void delete(String application, String entityToken, String sensorName) {
         final Entity entity = brooklyn().getEntity(application, entityToken);
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.MODIFY_ENTITY, entity)) {
```

Comment:
Agreed long-term, but don't think I'll address it here. Adding more types like that would also make it a bit more verbose to configure, unless we do some kind of hierarchy (e.g. would enabling `entity.modify` mean that `config.set` is implicitly enabled, unless it is explicitly overridden).